docs/openapi_v1.0.0-alpha.yaml

```
@@ -347,15 +367,20 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
-    put:
+
+  /suppliers/{supplierId}/bookings/{uuid}/cancel/confirm:
```

Nesting actions like this sets a bad pattern. Let's shorten this to a single action.

```suggestion
  /suppliers/{supplierId}/bookings/{uuid}/confirmCancel:
```